### ISSUE TITLE
Latest version of Chrome breaks React Developer Tools

### ISSUE THREAD

**Issue by michaelcuneo** [facebook/react-devtools#1242]
Just had all of my systems update Chrome over night to Version 70.0.3538.110 (Official Build) (64-bit)

Since the update I'm getting issues in my projects that are fixed by disabling the React Developer Tools extension... 

![screen shot 2018-11-30 at 3 17 05 pm](https://user-images.githubusercontent.com/12700009/49268591-67f1f180-f4b4-11e8-81a4-c3fa3d1a383b.png)

**Comment by bvaughn** [comment]
I am running version 70.0.3538.77 on macOS 10.13.6 and haven't seen any problem. I also haven't heard of any other than this issue. So I would guess it's probably not a Chrome+DevTools problem, but maybe something specific to the project you're seeing the error in?

Looking at the screenshot you've pasted, it looks like the errors are all coming from redux. What makes you suspect React DevTools is the cause?

Is this code running anywhere that I could take a look (in my browser, with DevTools installed) to see if I can repro?

**Comment by michaelcuneo** [comment]
Not sure what's occurring, but it is working 100% fine now.

I was using a fresh install of React-Boilerplate across all my environments OS's, Devices, etc... I tested on Chrome on Mac, Windows 10, and Ubuntu.

Each OS was running Chrome Version 70.0.3538.110 (Official Build) (64-bit), and React Developer Tools was 3.4.2 ...

It appears as though I am now running React Developer Tools 3.4.3, and the problem has now gone. I don't know what has occurred, but it works now.

I initially thought it was a problem with React-Boilerplate I posted an issue up there on their GitHub, while running through tests I realised that if I ran React-Boilerplate in Firefox, or Safari it worked 100% fine... 

So I realised that it was a problem with Chromes extensions. So I disabled each extension until the React-Boilerplate gave no errors... and the fix was disabling React Dev Tools.
